packages/react-components/src/components/DevicesButton.tsx

```diff
@@ -172,12 +172,28 @@ const generateDefaultMenuProps = (
 
   const defaultMenuProps: IContextualMenuProps = {
     items: [],
-
     styles: props.styles?.menuStyles,
-
-    // Confine the menu to the parents bounds.
-    // More info: https://github.com/microsoft/fluentui/issues/18835
-    calloutProps: { styles: { root: { maxWidth: '95%' } } }
+    calloutProps: {
+      styles: {
+        root: {
+          // Confine the menu to the parents bounds.
+          // More info: https://github.com/microsoft/fluentui/issues/18835
+          maxWidth: '100%'
```

Comment:
keep as 95% here (and probably 95% n the others really), because the size of the actual callout is slightly bigger (drop shadow and some margins) than what this calloutProps targets

Reply:
(You can git blame me on that one for not adding a good comment 😳)

Reply:
Thanks for catching that. Also added comment.